R/versions.R

```diff
@@ -169,7 +169,7 @@ atime_versions_exprs <- function(pkg.path, expr, sha.vec=NULL, verbose=FALSE, pk
   a.args
 }
 
-atime_pkg <- function(pkg.path="."){
+atime_pkg <- function(pkg.path=".", tests.R.path=file.path("inst", "atime", "tests.R"))){
```

Comment:
even tests.dir would be enough, it could be nice to require ${dir}/atime/tests.R for consistency.

Reply:
ok I updated, is that what you meant?

Reply:
Yep, looks great.